ci/testkomodo.sh

```diff
@@ -49,8 +49,17 @@ start_tests () {
     export ECL_SKIP_SIGNAL=ON
 
     pushd ${CI_TEST_ROOT}/tests
+
     python -m pytest -n 4 --mpl --benchmark-disable --eclipse-simulator \
         --durations=0 -sv --dist loadgroup
+
+    mkdir -p pytest-tmp  # NFS mapped tmp directory
+
+    # Using presence of "bsub" in PATH to detect onprem vs azure
+    which bsub >/dev/null && basetemp=$(mktemp -d -p ~/pytest-tmp) \
+        pytest -sv --lsf --basetemp=$basetemp integration_tests/scheduler/test_lsf_driver.py && \
```

Comment:
Are we then running these `test_lsf_driver.py` tests 2x? Maybe it's o thoug.

Reply:
We are running them once through the regular pytest where it will use the mocked LSF queue, and then we rerun the same tests here toward the real LSF cluster. We are not running them twice towards the real LSF cluster.

Reply:
Right, I thought as a part of komodo tests it would run twice; ie. towards mocked and real LSF.

Reply:
It will after this PR :)